modules/bitgo/example/ts/avaxp/multisig-crosschain-export-import.ts

```diff
@@ -0,0 +1,250 @@
+import { Avalanche, BinTools, BN, Buffer } from 'avalanche';
+import {
+  AVMAPI,
+  KeyChain as AVMKeyChain,
+  AmountOutput,
+  UnsignedTx,
+  ExportTx,
+  TransferableOutput,
+  TransferableInput,
+  Tx as AVMTx,
+} from 'avalanche/dist/apis/avm';
+import { PlatformVMAPI,
+  SECPTransferOutput,
+  SECPTransferInput,
+  UTXOSet,
+  UTXO,
+  UnsignedTx as PVMUnsignedTx,
+  KeyChain as PlatformVMKeyChain,
+  Tx as PlatformVMTx,
+} from 'avalanche/dist/apis/platformvm';
+import {
+  Defaults,
+  UnixNow,
+} from 'avalanche/dist/utils';
+
+type ChainConfig = {
+  NODE_HOST: string;
+  NODE_PORT: number;
+  NODE_PROTOCOL: string;
+  NODE_NETWORK_ID: number;
+};
+
+const FUJI: ChainConfig = {
+  NODE_HOST: 'localhost',
+  NODE_PORT: 9650,
+  NODE_PROTOCOL: 'http',
+  NODE_NETWORK_ID: 5,
+};
+
+// const MAINNET: ChainConfig = {
+//   NODE_HOST: 'api.avax.network',
+//   NODE_PORT: 9650,
+//   NODE_PROTOCOL: 'http',
+//   NODE_NETWORK_ID: 1,
+// };
+
+// Fuji or Mainnet config
+const conf = FUJI;
+
+// Private Key containing funds
+const avaxPrivateKey = 'PrivateKey-e2PZP3FYBU3G3gkRXXZkYPtye2ZqFrbSA13uDiutLK5ercixX';
+
+// multisig wallet receive addresses
+const pMultisigReceiveAddresses = [
+  'P-fuji1g9xy3vv537dncm2438t2kpk3jq3rm9sfaamcsp',
+  'P-fuji1cgygucxnhe3daq8wh7n5557qf8t6gy5ravugxs',
+  'P-fuji1d0ynjkgjaranlztj7swl04awtxeksl5wpd2puz',
```

Comment:
are we interested in remove the specific value but leaving example like 
`PrivateKey-here`
`P-address1`
`P-address2`